### ISSUE TITLE
inconsistency with the disconnect app button

### ISSUE THREAD

**Issue by SomayChauhan** [calcom/cal.com#14307]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

**issue 1** - Despite installing the app on all available teams and your personal account, the "install" button persists on the appCard, while the app detail page shows the "disconnect" option.

**issue 2** - the disconnect app only removes the app from my personal account, there is no way to remove the app from a team.
It should ideally provide the option to select the account or team, similar to the installation process.

### Evidence

[LOOM](https://www.loom.com/share/c9be89a08ee34d47bfd327998bc61426?sid=d22335a6-66a8-4d86-bbba-a46f67b905ad)

**Comment by MuhmdAjeer** [comment]
@ciaranha can i work on this

**Comment by ciaranha** [comment]
> @ciaranha can i work on this

Yes, go ahead. No need to ask, anyone can take on an issue. Just mention this issue in your pr and when it's ready for review someone from the team will take a look  v and review. Thanks 🙏

**Comment by hariombalhara** [comment]
The linked PR is fixing only issue-2. issue-1 can be fixed in another PR.
